index.html

```diff
@@ -0,0 +1,815 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Space Ripples Reveal Big Bang’s Smoking Gun - The New York Times</title>
+</head>
+
+<body>
+    <header>
+        <nav class="navbar flex-justify-sb">
+            <div class="nav-container flex-justify-sb">
+                <a class="nav-link soft-black" href="#">
+                    <i class="fa fa-bars"></i>
+                </a>
+                <a class="nav-link margin-menu soft-black" href="#">
+                    <i class="fa fa-search"></i>
+                </a>
+                <a class="nav-link margin-menu" href="#">
+                    <h3 class="nyt-f fw-700 black">SPACE & COSMOS</h3>
+                </a>
+            </div>
+            <div class="nav-container">
+                <img src="./img/nyt-logo.png" alt="NYT Logo">
```

Comment:
The NY Times logo should be linked. i.e. use the `<a>` tag around the logo. (<a href="https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0" rel="nofollow noreferrer noopener" target="_blank">link to page</a>)

Reply:
Ready, thanks.